charts/celeborn/values.yaml

```diff
@@ -136,3 +136,38 @@ affinity:
 tolerations: []
 
 nodeSelector: {}
+
+# With minimal running cost, the default mount is emptyDir.
+# If users want to use the services of a local disk or cloud vendor,
+# should change the value
+volumes:
+  master:
+    - name: celeborn-master-volume
+      emptyDir:
+        sizeLimit: 1Gi
+  worker:
+    - name: celeborn-worker-vol-0
+      emptyDir:
+        sizeLimit: 2Gi
+    - name: celeborn-worker-vol-1
+      emptyDir:
+        sizeLimit: 2Gi
+    - name: celeborn-worker-vol-2
+      emptyDir:
+        sizeLimit: 2Gi
+
+# When changing the mount point,
+# you need to change the corresponding configuration address like:
+# celeborn.ha.master.ratis.raft.server.storage.dir
+# celeborn.worker.storage.dirs
+volumeMounts:
+  master:
+    - name: celeborn-master-volume
```

Comment:
The changes look good to me, and it would be better if CelebornConf(storage.dir/ratis dir) can modify accordingly

Reply:
Sure, i'm working on this